### ISSUE TITLE
add semantic skeleton

### ISSUE THREAD

**Issue by pkra** [zorkow/speech-rule-engine#243]
Cf. https://github.com/zorkow/AIM-Workshop/issues/16.

What would be needed to merge the aim-hackathon branch?

**Comment by pkra** [comment]
On a related note, I'm unable to compile that branch.

**Comment by zorkow** [comment]
Compiles fine for me. What is the error message?

**Comment by pkra** [comment]
Partial log:

```

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:416: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 4 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:420: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 4 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:425: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 4 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:429: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 4 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:437: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 4 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:443: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 4 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:448: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 6 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:454: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 6 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:460: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 4 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:466: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 4 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:475: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 7 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:481: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 7 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:486: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 10 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

./speech-rule-engine/src/speech_rules/semantic_tree_rules.js:493: ERROR - Function sre.SemanticTreeRules.defineRule_: called with 10 argument(s). Function requires at least 5 argument(s) and no more than 5 argument(s).
  defineRule(
  ^^^^^^^^^^^

498 error(s), 125 warning(s), 97.3% typed
Makefile:109: recipe for target './speech-rule-engine/lib/sre.js' failed
make: *** [./speech-rule-engine/lib/sre.js] Error 127
```

**Comment by pkra** [comment]
I cloned, checked out the branch, ran `npm i` (and then ran `make` seperately again).

I'm on node 10.

**Comment by zorkow** [comment]
Odd. Is that a clean build? I.e., did you do `make clean` first. Btw. the warnings are due to the new closure version. I have not upgraded the Makefile yet.

**Comment by zorkow** [comment]
OK, overlapped. Can you give me the closure version you are using?

**Comment by pkra** [comment]
The following package-lock.json was generated (after I removed the prepublish scripts anyway -- that's deprecated btw.)

```json
{
  "name": "speech-rule-engine",
  "version": "2.2.1",
  "lockfileVersion": 1,
  "requires": true,
  "dependencies": {
    "ansi-regex": {
      "version": "2.1.1",
      "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
      "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
      "dev": true
    },
    "ansi-styles": {
      "version": "2.2.1",
      "resolved": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-2.2.1.tgz",
      "integrity": "sha1-tDLdM1i2NM914eRmQ2gkBTPB3b4=",
      "dev": true
    },
    "babylon": {
      "version": "7.0.0-beta.19",
      "resolved": "https://registry.npmjs.org/babylon/-/babylon-7.0.0-beta.19.tgz",
      "integrity": "sha512-Vg0C9s/REX6/WIXN37UKpv5ZhRi6A4pjHlpkE34+8/a6c2W1Q692n3hmc+SZG5lKRnaExLUbxtJ1SVT+KaCQ/A==",
      "dev": true
    },
    "bluebird": {
      "version": "3.5.1",
      "resolved": "https://registry.npmjs.org/bluebird/-/bluebird-3.5.1.tgz",
      "integrity": "sha512-MKiLiV+I1AA596t9w1sQJ8jkiSr5+ZKi0WKrYGUn6d1Fx+Ij4tIj+m2WMQSGczs5jZVxV339chE8iwk6F64wjA==",
      "dev": true
    },
    "catharsis": {
      "version": "0.8.9",
      "resolved": "https://registry.npmjs.org/catharsis/-/catharsis-0.8.9.tgz",
      "integrity": "sha1-mMyJDKZS3S7w5ws3klMQ/56Q/Is=",
      "dev": true,
      "requires": {
        "underscore-contrib": "~0.3.0"
      }
    },
    "chalk": {
      "version": "1.1.3",
      "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
      "integrity": "sha1-qBFcVeSnAv5NFQq9OHKCKn4J/Jg=",
      "dev": true,
      "requires": {
        "ansi-styles": "^2.2.1",
        "escape-string-regexp": "^1.0.2",
        "has-ansi": "^2.0.0",
        "strip-ansi": "^3.0.0",
        "supports-color": "^2.0.0"
      }
    },
    "clone": {
      "version": "2.1.1",
      "resolved": "https://registry.npmjs.org/clone/-/clone-2.1.1.tgz",
      "integrity": "sha1-0hfR6WERjjrJpLi7oyhVU79kfNs=",
      "dev": true
    },
    "clone-buffer": {
      "version": "1.0.0",
      "resolved": "https://registry.npmjs.org/clone-buffer/-/clone-buffer-1.0.0.tgz",
      "integrity": "sha1-4+JbIHrE5wGvch4staFnksrD3Fg=",
      "dev": true
    },
    "clone-stats": {
      "version": "1.0.0",
      "resolved": "https://registry.npmjs.org/clone-stats/-/clone-stats-1.0.0.tgz",
      "integrity": "sha1-s3gt/4u1R04Yuba/D9/ngvh3doA=",
      "dev": true
    },
    "cloneable-readable": {
      "version": "1.1.2",
      "resolved": "https://registry.npmjs.org/cloneable-readable/-/cloneable-readable-1.1.2.tgz",
      "integrity": "sha512-Bq6+4t+lbM8vhTs/Bef5c5AdEMtapp/iFb6+s4/Hh9MVTt8OLKH7ZOOZSCT+Ys7hsHvqv0GuMPJ1lnQJVHvxpg==",
      "dev": true,
      "requires": {
        "inherits": "^2.0.1",
        "process-nextick-args": "^2.0.0",
        "readable-stream": "^2.3.5"
      }
    },
    "closure-linter-wrapper": {
      "version": "1.1.0",
      "resolved": "https://registry.npmjs.org/closure-linter-wrapper/-/closure-linter-wrapper-1.1.0.tgz",
      "integrity": "sha1-1lTNhU56yMqhQNDKSMy/MChUsSg=",
      "dev": true,
      "requires": {
        "colors": "^1.1.2"
      }
    },
    "colors": {
      "version": "1.3.0",
      "resolved": "https://registry.npmjs.org/colors/-/colors-1.3.0.tgz",
      "integrity": "sha512-EDpX3a7wHMWFA7PUHWPHNWqOxIIRSJetuwl0AS5Oi/5FMV8kWm69RTlgm00GKjBO1xFHMtBbL49yRtMMdticBw==",
      "dev": true
    },
    "commander": {
      "version": "2.15.1",
      "resolved": "https://registry.npmjs.org/commander/-/commander-2.15.1.tgz",
      "integrity": "sha512-VlfT9F3V0v+jr4yxPc5gg9s62/fIVWsd2Bk2iD435um1NlGMYdVCq+MjcXnhYq2icNOizHr1kK+5TI6H0Hy0ag=="
    },
    "core-util-is": {
      "version": "1.0.2",
      "resolved": "https://registry.npmjs.org/core-util-is/-/core-util-is-1.0.2.tgz",
      "integrity": "sha1-tf1UIgqivFq1eqtxQMlAdUUDwac=",
      "dev": true
    },
    "escape-string-regexp": {
      "version": "1.0.5",
      "resolved": "https://registry.npmjs.org/escape-string-regexp/-/escape-string-regexp-1.0.5.tgz",
      "integrity": "sha1-G2HAViGQqN/2rjuyzwIAyhMLhtQ=",
      "dev": true
    },
    "google-closure-compiler": {
      "version": "20180610.0.2",
      "resolved": "https://registry.npmjs.org/google-closure-compiler/-/google-closure-compiler-20180610.0.2.tgz",
      "integrity": "sha1-Eggy9gzK2ZXo5HxedRIRLVTGro8=",
      "dev": true,
      "requires": {
        "chalk": "^1.0.0",
        "vinyl": "^2.0.1",
        "vinyl-sourcemaps-apply": "^0.2.0"
      }
    },
    "google-closure-library": {
      "version": "20180506.0.0",
      "resolved": "https://registry.npmjs.org/google-closure-library/-/google-closure-library-20180506.0.0.tgz",
      "integrity": "sha1-LS2pLCfBDcADE//4ClFYtVTKmQg=",
      "dev": true
    },
    "graceful-fs": {
      "version": "4.1.11",
      "resolved": "https://registry.npmjs.org/graceful-fs/-/graceful-fs-4.1.11.tgz",
      "integrity": "sha1-Dovf5NHduIVNZOBOp8AOKgJuVlg=",
      "dev": true
    },
    "has-ansi": {
      "version": "2.0.0",
      "resolved": "https://registry.npmjs.org/has-ansi/-/has-ansi-2.0.0.tgz",
      "integrity": "sha1-NPUEnOHs3ysGSa8+8k5F7TVBbZE=",
      "dev": true,
      "requires": {
        "ansi-regex": "^2.0.0"
      }
    },
    "inherits": {
      "version": "2.0.3",
      "resolved": "https://registry.npmjs.org/inherits/-/inherits-2.0.3.tgz",
      "integrity": "sha1-Yzwsg+PaQqUC9SRmAiSA9CCCYd4=",
      "dev": true
    },
    "isarray": {
      "version": "1.0.0",
      "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
      "integrity": "sha1-u5NdSFgsuhaMBoNJV6VKPgcSTxE=",
      "dev": true
    },
    "js2xmlparser": {
      "version": "3.0.0",
      "resolved": "https://registry.npmjs.org/js2xmlparser/-/js2xmlparser-3.0.0.tgz",
      "integrity": "sha1-P7YOqgicVED5MZ9RdgzNB+JJlzM=",
      "dev": true,
      "requires": {
        "xmlcreate": "^1.0.1"
      }
    },
    "jsdoc": {
      "version": "3.5.5",
      "resolved": "https://registry.npmjs.org/jsdoc/-/jsdoc-3.5.5.tgz",
      "integrity": "sha512-6PxB65TAU4WO0Wzyr/4/YhlGovXl0EVYfpKbpSroSj0qBxT4/xod/l40Opkm38dRHRdQgdeY836M0uVnJQG7kg==",
      "dev": true,
      "requires": {
        "babylon": "7.0.0-beta.19",
        "bluebird": "~3.5.0",
        "catharsis": "~0.8.9",
        "escape-string-regexp": "~1.0.5",
        "js2xmlparser": "~3.0.0",
        "klaw": "~2.0.0",
        "marked": "~0.3.6",
        "mkdirp": "~0.5.1",
        "requizzle": "~0.2.1",
        "strip-json-comments": "~2.0.1",
        "taffydb": "2.6.2",
        "underscore": "~1.8.3"
      }
    },
    "klaw": {
      "version": "2.0.0",
      "resolved": "https://registry.npmjs.org/klaw/-/klaw-2.0.0.tgz",
      "integrity": "sha1-WcEo4Nxc5BAgEVEZTuucv4WGUPY=",
      "dev": true,
      "requires": {
        "graceful-fs": "^4.1.9"
      }
    },
    "marked": {
      "version": "0.3.19",
      "resolved": "https://registry.npmjs.org/marked/-/marked-0.3.19.tgz",
      "integrity": "sha512-ea2eGWOqNxPcXv8dyERdSr/6FmzvWwzjMxpfGB/sbMccXoct+xY+YukPD+QTUZwyvK7BZwcr4m21WBOW41pAkg==",
      "dev": true
    },
    "minimist": {
      "version": "0.0.8",
      "resolved": "https://registry.npmjs.org/minimist/-/minimist-0.0.8.tgz",
      "integrity": "sha1-hX/Kv8M5fSYluCKCYuhqp6ARsF0=",
      "dev": true
    },
    "mkdirp": {
      "version": "0.5.1",
      "resolved": "https://registry.npmjs.org/mkdirp/-/mkdirp-0.5.1.tgz",
      "integrity": "sha1-MAV0OOrGz3+MR2fzhkjWaX11yQM=",
      "dev": true,
      "requires": {
        "minimist": "0.0.8"
      }
    },
    "process-nextick-args": {
      "version": "2.0.0",
      "resolved": "https://registry.npmjs.org/process-nextick-args/-/process-nextick-args-2.0.0.tgz",
      "integrity": "sha512-MtEC1TqN0EU5nephaJ4rAtThHtC86dNN9qCuEhtshvpVBkAW5ZO7BASN9REnF9eoXGcRub+pFuKEpOHE+HbEMw==",
      "dev": true
    },
    "readable-stream": {
      "version": "2.3.6",
      "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
      "integrity": "sha512-tQtKA9WIAhBF3+VLAseyMqZeBjW0AHJoxOtYqSUZNJxauErmLbVm2FW1y+J/YA9dUrAC39ITejlZWhVIwawkKw==",
      "dev": true,
      "requires": {
        "core-util-is": "~1.0.0",
        "inherits": "~2.0.3",
        "isarray": "~1.0.0",
        "process-nextick-args": "~2.0.0",
        "safe-buffer": "~5.1.1",
        "string_decoder": "~1.1.1",
        "util-deprecate": "~1.0.1"
      }
    },
    "remove-trailing-separator": {
      "version": "1.1.0",
      "resolved": "https://registry.npmjs.org/remove-trailing-separator/-/remove-trailing-separator-1.1.0.tgz",
      "integrity": "sha1-wkvOKig62tW8P1jg1IJJuSN52O8=",
      "dev": true
    },
    "replace-ext": {
      "version": "1.0.0",
      "resolved": "https://registry.npmjs.org/replace-ext/-/replace-ext-1.0.0.tgz",
      "integrity": "sha1-3mMSg3P8v3w8z6TeWkgMRaZ5WOs=",
      "dev": true
    },
    "requizzle": {
      "version": "0.2.1",
      "resolved": "https://registry.npmjs.org/requizzle/-/requizzle-0.2.1.tgz",
      "integrity": "sha1-aUPDUwxNmn5G8c3dUcFY/GcM294=",
      "dev": true,
      "requires": {
        "underscore": "~1.6.0"
      },
      "dependencies": {
        "underscore": {
          "version": "1.6.0",
          "resolved": "https://registry.npmjs.org/underscore/-/underscore-1.6.0.tgz",
          "integrity": "sha1-izixDKze9jM3uLJOT/htRa6lKag=",
          "dev": true
        }
      }
    },
    "safe-buffer": {
      "version": "5.1.2",
      "resolved": "https://registry.npmjs.org/safe-buffer/-/safe-buffer-5.1.2.tgz",
      "integrity": "sha512-Gd2UZBJDkXlY7GbJxfsE8/nvKkUEU1G38c1siN6QP6a9PT9MmHB8GnpscSmMJSoF8LOIrt8ud/wPtojys4G6+g==",
      "dev": true
    },
    "sax": {
      "version": "0.4.2",
      "resolved": "https://registry.npmjs.org/sax/-/sax-0.4.2.tgz",
      "integrity": "sha1-OfO2AXM9a+yXEFskKipA/Wl4rDw="
    },
    "source-map": {
      "version": "0.5.7",
      "resolved": "https://registry.npmjs.org/source-map/-/source-map-0.5.7.tgz",
      "integrity": "sha1-igOdLRAh0i0eoUyA2OpGi6LvP8w=",
      "dev": true
    },
    "string_decoder": {
      "version": "1.1.1",
      "resolved": "https://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
      "integrity": "sha512-n/ShnvDi6FHbbVfviro+WojiFzv+s8MPMHBczVePfUpDJLwoLT0ht1l4YwBCbi8pJAveEEdnkHyPyTP/mzRfwg==",
      "dev": true,
      "requires": {
        "safe-buffer": "~5.1.0"
      }
    },
    "strip-ansi": {
      "version": "3.0.1",
      "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
      "integrity": "sha1-ajhfuIU9lS1f8F0Oiq+UJ43GPc8=",
      "dev": true,
      "requires": {
        "ansi-regex": "^2.0.0"
      }
    },
    "strip-json-comments": {
      "version": "2.0.1",
      "resolved": "https://registry.npmjs.org/strip-json-comments/-/strip-json-comments-2.0.1.tgz",
      "integrity": "sha1-PFMZQukIwml8DsNEhYwobHygpgo=",
      "dev": true
    },
    "supports-color": {
      "version": "2.0.0",
      "resolved": "https://registry.npmjs.org/supports-color/-/supports-color-2.0.0.tgz",
      "integrity": "sha1-U10EXOa2Nj+kARcIRimZXp3zJMc=",
      "dev": true
    },
    "taffydb": {
      "version": "2.6.2",
      "resolved": "https://registry.npmjs.org/taffydb/-/taffydb-2.6.2.tgz",
      "integrity": "sha1-fLy2S1oUG2ou/CxdLGe04VCyomg=",
      "dev": true
    },
    "underscore": {
      "version": "1.8.3",
      "resolved": "https://registry.npmjs.org/underscore/-/underscore-1.8.3.tgz",
      "integrity": "sha1-Tz+1OxBuYJf8+ctBCfKl6b36UCI=",
      "dev": true
    },
    "underscore-contrib": {
      "version": "0.3.0",
      "resolved": "https://registry.npmjs.org/underscore-contrib/-/underscore-contrib-0.3.0.tgz",
      "integrity": "sha1-ZltmwkeD+PorGMn4y7Dix9SMJsc=",
      "dev": true,
      "requires": {
        "underscore": "1.6.0"
      },
      "dependencies": {
        "underscore": {
          "version": "1.6.0",
          "resolved": "https://registry.npmjs.org/underscore/-/underscore-1.6.0.tgz",
          "integrity": "sha1-izixDKze9jM3uLJOT/htRa6lKag=",
          "dev": true
        }
      }
    },
    "util-deprecate": {
      "version": "1.0.2",
      "resolved": "https://registry.npmjs.org/util-deprecate/-/util-deprecate-1.0.2.tgz",
      "integrity": "sha1-RQ1Nyfpw3nMnYvvS1KKJgUGaDM8=",
      "dev": true
    },
    "vinyl": {
      "version": "2.1.0",
      "resolved": "https://registry.npmjs.org/vinyl/-/vinyl-2.1.0.tgz",
      "integrity": "sha1-Ah+cLPlR1rk5lDyJ617lrdT9kkw=",
      "dev": true,
      "requires": {
        "clone": "^2.1.1",
        "clone-buffer": "^1.0.0",
        "clone-stats": "^1.0.0",
        "cloneable-readable": "^1.0.0",
        "remove-trailing-separator": "^1.0.1",
        "replace-ext": "^1.0.0"
      }
    },
    "vinyl-sourcemaps-apply": {
      "version": "0.2.1",
      "resolved": "https://registry.npmjs.org/vinyl-sourcemaps-apply/-/vinyl-sourcemaps-apply-0.2.1.tgz",
      "integrity": "sha1-q2VJ1h0XLCsbh75cUI0jnI74dwU=",
      "dev": true,
      "requires": {
        "source-map": "^0.5.1"
      }
    },
    "wicked-good-xpath": {
      "version": "1.3.0",
      "resolved": "https://registry.npmjs.org/wicked-good-xpath/-/wicked-good-xpath-1.3.0.tgz",
      "integrity": "sha1-gbDpXoZQ5JyUsiKY//hoa1VTz2w="
    },
    "xml-mapping": {
      "version": "1.7.1",
      "resolved": "https://registry.npmjs.org/xml-mapping/-/xml-mapping-1.7.1.tgz",
      "integrity": "sha1-ZWiWWeUIWDPH0r7FfawoQsy7woY=",
      "requires": {
        "sax": "=0.4.2",
        "xml-writer": ">=1.0.4"
      }
    },
    "xml-writer": {
      "version": "1.7.0",
      "resolved": "https://registry.npmjs.org/xml-writer/-/xml-writer-1.7.0.tgz",
      "integrity": "sha1-t28dWRwWomNOvbcDx729D9aBkGU="
    },
    "xmlcreate": {
      "version": "1.0.2",
      "resolved": "https://registry.npmjs.org/xmlcreate/-/xmlcreate-1.0.2.tgz",
      "integrity": "sha1-+mv3YqYKQT+z3Y9LA8WyaSONMI8=",
      "dev": true
    },
    "xmldom-sre": {
      "version": "0.1.31",
      "resolved": "https://registry.npmjs.org/xmldom-sre/-/xmldom-sre-0.1.31.tgz",
      "integrity": "sha512-f9s+fUkX04BxQf+7mMWAp5zk61pciie+fFLC9hX9UVvCeJQfNHRHXpeo5MPcR0EUf57PYLdt+ZO4f3Ipk2oZUw=="
    }
  }
}
```

**Comment by zorkow** [comment]
The problem was introduced by closure's [June 10, 2018 (v20180610) release](https://github.com/google/closure-compiler/wiki/Releases#june-10-2018-v20180610). For no I've fixed the version to the previous release [v20180506](https://github.com/google/closure-compiler/wiki/Releases#may-06-2018-v20180506).

The 123 warnings were introduced by closure [v20180402](https://github.com/google/closure-compiler/wiki/Releases#may-06-2018-v20180506).

**Comment by zorkow** [comment]
@pkra Once I've fixed issue #245 and wrote/updated some tests it should land in the next version.
We would need a new descriptive attribute name of course as it is not the same as the `collapsed` attribute that records when we have removed layers in the semantic tree during enrichment.

**Comment by pkra** [comment]
Thanks, this fixed it (note to self: the fix is on master aim-hackathon).

**Comment by pkra** [comment]
This issues was actually for tracking the merge of the skeleton code, not the build issue ;-)

**Comment by zorkow** [comment]
OK. Github has closed the issue automatically after the merge.

**Comment by pkra** [comment]
I believe this has been resolved in 2.2.2 -- thank you!
